ppml/docs/prepare_environment.md

```diff
@@ -0,0 +1,157 @@
+
+To use the KMS service in PPML, follow the document: https://github.com/intel-analytics/BigDL/blob/main/ppml/services/pccs-ehsm/kubernetes/README.md
+
+### Attestation Service (AS) Setup
+placeholder
+
+### Start BigDL PPML Client Container
+1. Prepare BigDL PPML Docker Image
+
+    Pull Docker image from Dockerhub
+    ```
+    docker pull intelanalytics/bigdl-ppml-trusted-big-data-ml-scala-graphene:devel
+    ```
+
+    Alternatively, you can build Docker image from Dockerfile (this will take some time):
+    ```
+    cd trusted-big-data-ml/python/docker-graphene
+    ./build-docker-image.sh
+    ```
+    
+    **Note:** The above docker image `intelanalytics/bigdl-ppml-trusted-big-data-ml-scala-graphene:devel` is only used for demo purposes. You are recommended to refer to the [BigDL PPML Dockerfile](https://github.com/intel-analytics/BigDL/blob/main/ppml/trusted-big-data-ml/python/docker-graphene/Dockerfile) or use BigDL PPML image as a base image to build your own image and sign your image with your own enclave_key.
+    
+2. Start BigDL PPML client container
+    
+    Configure the environment variables in the following script before running it.
+    ```
+    export K8S_MASTER=k8s://$(sudo kubectl cluster-info | grep 'https.*6443' -o -m 1)
+    echo The k8s master is $K8S_MASTER .
+    export ENCLAVE_KEY=/YOUR_DIR/enclave-key.pem
+    export DATA_PATH=/YOUR_DIR/data
+    export KEYS_PATH=/YOUR_DIR/keys
+    export SECURE_PASSWORD_PATH=/YOUR_DIR/password
+    export KUBECONFIG_PATH=/YOUR_DIR/kubeconfig
+    export LOCAL_IP=$LOCAL_IP
+    export DOCKER_IMAGE=intelanalytics/bigdl-ppml-trusted-big-data-ml-python-graphene:devel
+    sudo docker run -itd \
+        --privileged \
+        --net=host \
+        --name=bigdl-ppml-client-k8s \
+        --cpuset-cpus="0-4" \
+        --oom-kill-disable \
+        --device=/dev/sgx/enclave \
+        --device=/dev/sgx/provision \
+        -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+        -v $ENCLAVE_KEY:/graphene/Pal/src/host/Linux-SGX/signer/enclave-key.pem \
+        -v $DATA_PATH:/ppml/trusted-big-data-ml/work/data \
+        -v $KEYS_PATH:/ppml/trusted-big-data-ml/work/keys \
+        -v $SECURE_PASSWORD_PATH:/ppml/trusted-big-data-ml/work/password \
+        -v $KUBECONFIG_PATH:/root/.kube/config \
+        -e RUNTIME_SPARK_MASTER=$K8S_MASTER \
+        -e RUNTIME_K8S_SERVICE_ACCOUNT=spark \
+        -e RUNTIME_K8S_SPARK_IMAGE=$DOCKER_IMAGE \
+        -e RUNTIME_DRIVER_HOST=$LOCAL_IP \
+        -e RUNTIME_DRIVER_PORT=54321 \
+        -e RUNTIME_DRIVER_CORES=1 \
+        -e RUNTIME_EXECUTOR_INSTANCES=1 \
+        -e RUNTIME_EXECUTOR_CORES=8 \
+        -e RUNTIME_EXECUTOR_MEMORY=1g \
+        -e RUNTIME_TOTAL_EXECUTOR_CORES=4 \
```

Comment:
this line need to be removed. we use RUNTIME_EXECUTOR_INSTANCES and RUNTIME_EXECUTOR_CORES to determine RUNTIME_TOTAL_EXECUTOR_CORES

Reply:
thx! removed.